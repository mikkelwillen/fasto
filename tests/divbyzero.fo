fun int main() =
    let n = 2 / 0 in
    write(n)