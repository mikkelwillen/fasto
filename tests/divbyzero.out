Runtime error at line 2: division by zero